### ISSUE TITLE
Version prefix not applying to current version in plugins

### ISSUE THREAD

**Issue by srt4rulez** [markchalloner/git-semver#31]
Using the example plugin without modification, the "current version" argument doesn't contain VERSION_PREFIX config setting.

See output:

```
$ git semver minor

User plugin "example.sh":

  New version: v0.2.0
  Current version: 0.1.0
  ...
v0.2.0
```

Notice how "current version" is "0.1.0" while new version is "v0.2.0" (with v)

This seems to be because the `version-do` method is being called without the VERSON_PREFIX for `$version` in `version-major`, `version-minor`, and `version-patch` :

https://github.com/markchalloner/git-semver/blob/d838cb34780a29d104ed20ef18465c44918b0f64/git-semver.sh#L165

I can make a PR.

**Comment by markchalloner** [comment]
Thanks for the report. Looking at the code it seems like [version-get's return](https://github.com/markchalloner/git-semver/blob/d838cb34780a29d104ed20ef18465c44918b0f64/git-semver.sh#L150) would be the natural place to prefix this.

> I can make a PR.

Yes please.

**Comment by srt4rulez** [comment]
Right, that was my first thought as well, but then the checks for a blank version in patch, minor, and major will possibly never be true (if a version prefix is set)

https://github.com/markchalloner/git-semver/blob/d838cb34780a29d104ed20ef18465c44918b0f64/git-semver.sh#L156-L161

Unless we strip off the version prefix for that check?

**Comment by markchalloner** [comment]
@srt4rulez the checks would still be valid, [version-get returns without echoing](https://github.com/markchalloner/git-semver/blob/d838cb34780a29d104ed20ef18465c44918b0f64/git-semver.sh#L148) if it can't parse out a valid version.

**Comment by srt4rulez** [comment]
Yes, I noticed that after doing some tests. I see you pulled in #27, which has some major changes.

**Comment by srt4rulez** [comment]
This seems to be fixed from #33.
